support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.jsx

```diff
@@ -113,7 +113,7 @@ class StripeForm extends Component<StripeFormPropTypes, StateTypes> {
     // is handled in the callback below by checking the value of paymentWaiting.
     fetchJson(
       this.props.stripeSetupIntentEndpoint,
-      requestOptions({ stripePublicKey: this.props.stripeKey }, 'omit', 'POST', this.props.csrf),
+      requestOptions({ stripePublicKey: this.props.stripeKey },  'same-origin','POST', this.props.csrf),
```

Comment:
This is needed because otherwise the cookies don't get sent over the AJAX request?

Reply:
yep, took me a while to figure that one out!